.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Default development database
 devdata.db
+devdata.db-journal
```

Comment:
Not sure when this file gets created... have not seen on my system? Are `-journal` files specific to your system (if so, add to your own global `.gitignore`, not ours)

Reply:
These are created when SQLite is doing operations. Sometimes it doesn't clean up properly, so this is necessary.